openapi.yml

```diff
@@ -6,23 +6,46 @@ servers:
 - url: http://localhost:8080
   description: Local
 paths:
-  /api/v1/order:
+  /api/v1/orders:
```

Comment:
Will we be adding a `GET   /api/v1/orders/{ID}` endpoint?

Reply:
I added the GET endpoint. The Delete is more complicated and we should add it later at some point. I am not sure how the authentication should work. Probably, one must have a specific signed message with the order digest.

Reply:
I imagine something like:
```
signTypedData({
    operation: "delete",
    orderUid: "0x...",
})
```

The service could then verify that
```
ecrecover(message) == order.owner
```

And return a `403: Forbidden` if the check fails.